.github/docker/Dockerfile-musllinux.template

```diff
@@ -40,10 +42,14 @@ RUN apk add -u \
 # exist here.
   cd /opt/_internal && \
   tar -Jxvf static-libs-for-embedding-only.tar.xz && \
+# Change required version of pydantic to be <2
+  sed 's/^\(pydantic>.*<\).*$/\12/' -i /tmp/requirements_full.txt && \
```

Comment:
It already is set to be `<2` in the file it seems? So is this necessary?

Reply:
DLite itself supports both pydantic v1 and v2. It is just manylinux-2010 and musllinux_1_1 that doesn't, because of pydantic v2 requires when installing on these platforms.

Requiring v1 doesn't affect the pip package. It only means that we will be testing the wheel against Pydantic v1, which is the sensible thing to do when v2 isn't easily supported on the older versions we are building for.